<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuscarMedico</class>
 <widget class="QMainWindow" name="BuscarMedico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1150</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buscar médico</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>1061</width>
      <height>651</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Médicos</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tablaMedicos">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>641</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Paterno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellido Materno</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cedula</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Turno</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="botonBuscar">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>330</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar a Todos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por cedula</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCedula">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarCedula">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputIdMedico">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarId">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>120</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputNombre">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>170</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarNombre">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>170</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por apellido</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputApellido">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarApellido">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>220</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar por turno</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputTurno">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>270</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buscarTurno">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>270</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menú</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
  <action name="actionID">
   <property name="text">
    <string>ID</string>
   </property>
  </action>
  <action name="actionNombre">
   <property name="text">
    <string>Nombre</string>
   </property>
  </action>
  <action name="actionCedula">
   <property name="text">
    <string>Cedula</string>
   </property>
  </action>
  <action name="actionTurno">
   <property name="text">
    <string>Turno</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
